core/converters.go

```diff
@@ -25,15 +24,6 @@ func ConvertBytes(bytes uint64) string {
 	return fmt.Sprintf("%.2f GB", float64(bytes)/1024.0/1024.0/1024.0)
 }
 
-// ToHex encodes the given buff to hex
-// This should be used only for display purposes!
-func ToHex(buff []byte) string {
-	if buff == nil {
-		return "<NIL>"
```

Comment:
if the buff was nil this one returned <NIL> while the hex.EncodeToString returns "". It seems that all tests pass.

Reply:
yes, was mostly used in prints and tests. Te reasoning behind removal of this function is that hex.Encode provides sufficient description and ToHex was a simple wrapper, no useful information

Reply:
ok